sites/partners/src/listings/PaperListingForm/formTypes.ts

```diff
@@ -92,7 +92,7 @@ export const formDefaults: FormListing = {
   applicationDueTime: null,
   applicationFee: null,
   applicationMethods: [],
-  applicationOpenDate: new Date(moment().subtract(10).format()),
+  applicationOpenDate: new Date(dayjs().subtract(10).format()),
```

Comment:
This was already an issue, but I think this code is missing the second argument so we're not sure what the unit should be. Subtract 10 seconds? Minutes? Days?

Reply:
I don't think we need this. If it's "now" that will still be in the past once it's saved.